docs/evacuation_patterns.md

```diff
@@ -0,0 +1,31 @@
+# Masakari's Evacuation Patterns
+
+Masakari starts to evacuate VM instances basing on instance, process and/or
+host failures monitored by each process. This documents describes patterns about
+the evacuations executed by masakari-controller.
+
+Each monitor notifies indivisual failures to controller based on what they
+monitor. You can choose which falures of instances are rescued by Masakari,
+deploying which monitoring process or not.
+
+## Evacuation patterns
+
+The section shows events that tigger Masakari to call which nova API.
+
+| Events | Monitored by | Previous instance's status | Rescue steps | Post instance's status |
+| :--- | :--- | :--- | :--- | :--- |
+| instance down | instancemonitor | active | nova.stop -> nova.start | active |
+| instance down | instancemonitor | stopped | nova.stop | stopped |
```

Comment:
"nova.stop" should be "nova.reset" in case of an instance down of 'stopped' status instance.
Optionally, adding the status note of nova.reset like nova.reset(stop) would be beneficial to understand.